ui/apps/platform/src/routePaths.ts

```diff
@@ -0,0 +1,315 @@
+        readAccessPredicate: renderUnconditionally,
+    },
+    [networkBasePath]: {
+        readAccessPredicate: readAccessResourceNames([]), // NetworkGraph, and NetworkPolicy?
+    },
+    [violationsBasePath]: {
+        readAccessPredicate: readAccessResourceNames([]), // Alert
+    },
+    [complianceBasePath]: {
+        readAccessPredicate: readAccessResourceNames([]), // Compliance
+    },
+
+    // Vulnerability Management
+    [vulnManagementPath]: {
+        readAccessPredicate: readAccessResourceNames([]),
+    },
+    [vulnManagementRiskAcceptancePath]: {
+        readAccessPredicate: readAccessResourceNames([]), // VulnerabilityManagementApprovals and/or VulnerabilityManagementRequests?
+    },
+    [vulnManagementReportsPath]: {
+        readAccessPredicate: readAccessResourceNames(['VulnerabilityReports']),
+    },
+
+    [configManagementPath]: {
+        readAccessPredicate: readAccessResourceNames([]),
+    },
+    [riskBasePath]: {
+        readAccessPredicate: readAccessResourceNames([]), // Deployment, and DeploymentExtension?
+    },
+
+    // Platform Configuration
+    [clustersBasePath]: {
+        readAccessPredicate: readAccessResourceNames([]), // Cluster
+    },
+    /*
+    [clustersListPath]: {
+        featureFlagDependency: 'ROX_TODO', // replace conditional development rendering with backend feature flag
+        readAccessPredicate: readAccessResourceNames([]), // Cluster
+    },
+    */
+    [policiesBasePath]: {
+        readAccessPredicate: readAccessResourceNames([]), // Policy
+    },
+    [integrationsPath]: {
+        readAccessPredicate: readAccessResourceNames([]), // AuthPlugin is obsolete? APIToken; BackupPlugins, ImageIntegration, Notifier, SignatureIntegration; superseded by Integration?
+    },
+    [accessControlBasePathV2]: {
+        readAccessPredicate: readAccessResourceNames([]), // Access
+    },
+    [systemConfigPath]: {
+        readAccessPredicate: readAccessResourceNames(['Config']),
+    },
+    [systemHealthPath]: {
+        readAccessPredicate: readAccessResourceNames([]),
+    },
+    [systemHealthPathPF]: {
+        featureFlagDependency: 'ROX_SYSTEM_HEALTH_PF',
+        readAccessPredicate: readAccessResourceNames([]),
+    },
+
+    // Header
+    [apidocsPath]: {
+        readAccessPredicate: readAccessResourceNames([]),
+    },
+    // Help Center is an external link to /docs/product
+    [userBasePath]: {
+        readAccessPredicate: readAccessResourceNames([]),
+    },
+};
+
+/*
+ * Evaluate feature flags and resource requirements for route paths.
+ */
+
+export type IsRenderedRoutePath = (routePath: RoutePath) => boolean;
+
+/*
+ * Higher-order function if caller needs to have predicate functions in its scope.
+ * For example, MainPath because:
+ * Body needs both isFeatureFlagEnabled and isRenderedRoutePath.
+ * NaviationSidebar needs only isRenderedRoutePath.
+ */
+export function getIsRenderedRoutePath(
```

Comment:
I was a little confused by the naming again here. Would it make more sense to say `getIsRoutePathRendered` and `isRoutePathRendered`?